src/App.js

```diff
@@ -387,14 +619,32 @@ function Stat({
           fColor={"#a93226"}
           increased={deadCountIncrease}
         >
-          Deaths
+        <HtmlTooltip
+          title={
+            <React.Fragment>
+              <Typography color="inherit">Deaths</Typography>
+              <em>{"All cases that resulted in death, mostly found in the elderly."}</em>
```

Comment:
This one is probably unnecessary.

Reply:
It's just there to make sure that any tag will have a definition. I find it awkward to have only one with a definition when hovered. But you guys can remove it if you want to. Just a personal pov.